### ISSUE TITLE
[Feature request] A way to re-run Steam Play installation scripts

### ISSUE THREAD

**Issue by cprn** [ValveSoftware/steam-for-linux#6345]
#### Your system information

* Steam client version (build number or date): Jun 21 2019, at 01:43:20
* Distribution (e.g. Ubuntu): Manjaro.i3
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
I selected a game -> right click -> "Uninstall..." and confirmed. Only I didn't notice my sticky shift made me select half of the library... Why is there no "cancel" button while accidentally uninstalling 97 games?

It was obviously my mistake but it took long time to uninstall all the games and then select them again and re-download 200 GB of installation files (Kingdom Come, Shadow of Mordor, etc).

#### Steps for reproducing this issue:

1. Install multiple games
2. Select multiple games
3. Uninstall them
4. Notice there's no way to stop the process

**Comment by kisak-valve** [comment]
Hello @cprn, the issue report title doesn't match the issue you've described. Can you clarify the intent of this issue report?

**Comment by cprn** [comment]
Yeah, sorry, I was opening 2 requests at once. Fixing right away.
